configs/defaults/rare_disease_rnaseq.toml

```diff
@@ -0,0 +1,5 @@
+[workflow]
+sequencing_type = "rna"
+
+[trim]
+adapter_type = "ILLUMINA_TRUSEQ"
```

Comment:
All default configs also need to be listed in cpg_workflows/defaults.toml with an explanation of what is being configured